### DISCUSSION TITLE
Python

### DISCUSSION THREAD

**Discussion by madiyorbekodilov** [community/community#102780] [Discussions]
### Select Topic Area

General

### Body

Is it possible to create an android program in python?

**Comment by muslimbek77** [comment]
Yes, you can make an Android app with Python. There are several tools available that allow you to use Python to create Android apps, such as Kivy, Pygame, and BeeWare. Each of these tools has its own advantages and disadvantages, so you should research them to determine which one is best for your project.

**Reply by ettaboyle** [reply]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by ettaboyle** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
